### ISSUE TITLE
Discover what component to do usability testing on

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds-site#1984]
USWDS aims to initiate regular usability testing of components, templates, patterns, and guidance and one of our roadmapping goals is to outline a model/process for how we go about carrying out testing. Toward that goal, we need to perform testing on components to help inform how we operationalize testing.

- [x] First we need to brainstorm components that need testing to create a backlog. 
- [x] Consult with @amyleadem who is working on the component lifecycle to see how this effort to choose what components to work on ties into the component lifecycle
- [x] Outline how we will prioritize components to test
- [x] Decide on components to test first

**Comment by jaclinec** [comment]
Based on a brainstorm meeting with the USWDS team as well as work from the Inclusive Patterns team, I've created a components testing backlog section that lives within the larger USWDS Research Practice Backlog. I've included some info on things to consider when prioritizing what we test. Such as:


Component testing prioritization is a flexible system that considers:

- Severity of known usability or accessibility problems
- Potential impact to users (# of users affected)
- Usage/popularity of components
- What components make sense to test together
- Components with no research (undocumented components and new components)
- Level of effort?

I have been reviewing the Inclusive Patterns team's User Acceptance Testing research where they tested the usability of several components and patterns for accessibility issues. I've included relevant insights from their research onto the same Mural board mentioned above. I recommend testing the components where the most severe usability/accessibility issues came up in testing.

**Comment by jaclinec** [comment]
At the last CS/UX sync, I presented possible criteria on how to prioritize what components to test, and showed the Inclusive Patterns team's recent UAT research which showed findings on the usability and accessibility of various components. We decided that we should prioritize testing the components with the most severe issues found and any relevant components that make sense to test together, including those where only minor issues were found. 

Components with the most severe issues found are: date picker, date range, and input mask.
